### ISSUE TITLE
Pre-generate USA spatial configuration

### ISSUE THREAD

**Issue by jcblemai** [HopkinsIDD/COVIDScenarioPipeline#85]
Maybe we want someone to write a script that takes USA wide ACS commute, CSSE cases and pop data to generate a `seeding.csv`, `geodata.csv` and `mobility.txt` for any state easily ? 

like `Rscript BuildSpatialSetup.R CA OR WA` or `Rscript BuildSpatialSetup.R USA -AK`.

It'll help if users have something functional to start with.

**Comment by jkamins7** [comment]
yeah, something like this is totally doable, but we haven't done it yet

**Comment by khwilson** [comment]
FWIW, I wrote this script that does the shapefile importation: https://gist.github.com/khwilson/2fc31a0a8b365790828020616bd157ba

**Comment by jkamins7** [comment]
@khwilson I think we'd like that as a pull request.  @jcblemai is there a good place in the repo for him to put it?

**Comment by jcblemai** [comment]
We should have a build-setup folder, with @khwilson script (very nice btw) and every (not deprecated) script we removed from data/, along with @jkamins7 USA script

**Comment by khwilson** [comment]
Happy to do a PR. Just let me know where to put it :)

**Comment by khwilson** [comment]
Just for reference, this is the setup that I have been using for Rhode Island at the moment. It's one big ol' docker jank command, but it does all the setup for you.

**Comment by jkamins7** [comment]
@khwilson we don't have a place for python scripts at the moment.  Can you create a top level folder called scripts and put it there?
